views/tunnel_setup.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Mozilla IoT Gateway Subdomain Creation</title>
```

Comment:
Nit: We changed all the other page titles to "Things Gateway" rather than "MozIoT Gateway" or "Mozilla IoT Gateway"